README.md

```diff
@@ -203,6 +203,39 @@ Notes:
 - If you are unable to use `uv tool install`, you can use `uvx cratedb-mcp`
   to acquire the package and run the application ephemerally.
 
+### Use OCI Standard
+
+OCI images for Docker or Podman are available per `ghcr.io/crate/cratedb-mcp`.
+See also [Docker Hub MCP Server] and [mcp hub].
+
+Probe invocation:
+```shell
+docker run --rm -it --entrypoint="" ghcr.io/crate/cratedb-mcp cratedb-mcp --version
+```
+
+Your command/args configuration snippet for the agent client user interface:
+```json
+{
+    "command": "docker",
+    "args": [
+      "run",
+      "--rm",
+      "-i",
+      "--env=CRATEDB_CLUSTER_URL",
+      "ghcr.io/crate/cratedb-mcp:latest"
+    ]
+}
+```
+
+### Use OCI MCPO
+
+For integrating Open WebUI, the project provides an OCI MCPO image which wraps
+the MCP server using the `mcpo` proxy. See also [MCP support for Open WebUI] and
+[MCP-to-OpenAPI proxy server (mcpo)].
+```shell
+docker run --rm --env=CRATEDB_CLUSTER_URL --publish 8000:8000 ghcr.io/crate/cratedb-mcpo
+```
```

Comment:
mcpo support for Open WebUI also has not been validated yet.

Reply:
The documentation sections about using the OCI images have been improved significantly with 81a357ff256.